runtime/jvmti/suspendhelper.cpp

```diff
@@ -27,7 +27,7 @@
 extern "C" {
 
 jvmtiError
-suspendThread(J9VMThread *currentThread, jthread thread, UDATA allowNull, UDATA *currentThreadSuspended)
+suspendThread(J9JavaVM *vm, J9VMThread *currentThread, jthread thread, UDATA allowNull, UDATA *currentThreadSuspended)
```

Comment:
VM can always be fetched from the current thread - there's no need to add the parameter.